### PR TITLE
Allow DNS incoming requests through UFW

### PR DESCRIPTION
**Pull request by EWouters**
Fixes https://github.com/pivpn/pivpn/issues/1282

Also added a line to the uninstall script to remove the rule at uninstall time.

### CODE REVIEW

**Comment by 4s3ti** [comment]
lgtm, Thanks a lot! 

Merging. 
I might merge test into master this weekend.

File: auto_install/install.sh [hunk 1]
```diff
@@ -1473,6 +1473,12 @@ askClientDNS(){
 
 			echo "pivpnDNS1=${pivpnDNS1}" >> ${tempsetupVarsFile}
 			echo "pivpnDNS2=${pivpnDNS2}" >> ${tempsetupVarsFile}
+
+			# Allow DNS incomming requests through UFW.
```
Comment (EWouters) [inline auto_install/install.sh:1477]: I messed up the spelling of the comment, I spelled incoming with two m's. I thought I fixed it but it is still written with two m's.

Comment (4s3ti) [reply auto_install/install.sh:1477]: hmm ... Nope its actually correct ... https://github.com/pivpn/pivpn/blob/test/auto_install/install.sh#L1477

That is probably older version

Comment (EWouters) [reply auto_install/install.sh:1477]: Strange, but fine :)

Comment (orazioedoardo) [reply auto_install/install.sh:1477]: @4s3ti @EWouters I’ve changed it, that this conversation is marked as resolved.

**Pull request merged**